### PR TITLE
Cockpit 179

### PR DESCRIPTION
**Pull request by Gundersanne**

### CODE REVIEW

**Comment by Gundersanne** [comment]
https://gundersanne.github.io/cockpit-project.github.io/blog/cockpit-179.html

File: _posts/2018-10-04-cockpit-179.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 179.
+
+### Machines: Detach disk support
+
+The Machines page now supports detaching a disk from VMs with the LibvirtDBus
+provider.
```
Comment (martinpitt) [inline _posts/2018-10-04-cockpit-179.md]: This is a bit misleading, as the provider doesn't apply to the VM itself. How about "When using the libvirt D-Bus provider, the  Machines page now supports detaching a disk from VMs."

File: _posts/2018-10-04-cockpit-179.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 179
+author: Gundersanne
+date: 2018-10-04
+tags: cockpit linux
+slug: cockpit-179
+category: release
+summary: Detach disk from VM with LibvirtDBus provider and offer cockpit-machines as Application
```
Comment (martinpitt) [inline _posts/2018-10-04-cockpit-179.md]: To be consistent with other releases: "Cockpit with Machines improvements"

File: _posts/2018-10-04-cockpit-179.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+The Machines page now supports detaching a disk from VMs with the LibvirtDBus
+provider.
+
+![Detach disk](/images/machines-detach-disk.png)
+
+_Thanks to Katerina Koukiou for this improvement!_
+
+### Machines: Offer cockpit-machines as Application
+
+This Cockpit version also adds AppStream metadata for its own `cockpit-machines`
```
Comment (martinpitt) [inline _posts/2018-10-04-cockpit-179.md]: Let's drop the "also"

**Pull request merged**